docs/tls/CERTIFICATES.md

```diff
@@ -0,0 +1,155 @@
+# Self-signed certificates guide
+
+This guide describes the steps required to generate self-signed certificates for use with NGINX Plus and NLK. 
+
+Using self-signed certificates is more secure than no TLS, but less secure than using certificates signed by a Certificate Authority (CA); 
+however, using self-signed certificates is easier to configure than using certificates signed by a CA. 
+The traffic between NLK and NGINX Plus will be encrypted, but each side will need to be configured to trust the certificates.
+
+When using self-signed certificates, the first step is to generate the Certificate Authority (CA),
+which will be used to sign the certificates for the client and the server.
+The `openssl` utility will be used to generate the CA and the certificates.
+
+#### Generate the CA
```

Comment:
lets call the CA a Certificate Authority, at least once

Reply:
dammit! everywhere I do! Thanks-a-lot ChatGPT ;-)